--- rvm-1.14.vanilla/rds/Makefile.am	2004-06-07 20:46:52.000000000 +0200
+++ rvm-1.14/rds/Makefile.am	2007-07-10 00:47:35.000000000 +0200
@@ -25,6 +25,7 @@
     rds_prealloc.c rds_split.c rds_stats.c rds_start.c rds_util.c rds_zap.c \
     rds_maxblock.c rds_private.h
 librds_la_LDFLAGS = -version-info $(LIBTOOL_VERSION) $(LIBTOOL_LDFLAGS)
+librds_la_LIBADD = $(top_builddir)/seg/libseg.la
 
 librdslwp_la_SOURCES = rds_coalesce_lwp.c rds_free_lwp.c rds_init_lwp.c \
     rds_malloc_lwp.c rds_prealloc_lwp.c rds_split_lwp.c rds_stats_lwp.c \
